client/extensions/woocommerce/app/settings/email/email-settings/index.js

```diff
@@ -2,38 +2,195 @@
+		subtitle: translate( 'Sent when an order is marked \'payment pending\'.' ),
+	},
+	{
+		field: 'email_customer_processing_order',
+		title: translate( 'Processing order' ),
+		subtitle: translate( 'Sent when an order is marked \'payment processing\'.' ),
+	},
+	{
+		field: 'email_customer_completed_order',
+		title: translate( 'Completed order' ),
+		subtitle: translate( 'Sent when an order is marked \'paid in full\'.' ),
+	},
+	{
+		field: 'email_customer_refunded_order',
+		title: translate( 'Refunded order' ),
+		subtitle: translate( 'Sent when an order is marked \'payment refunded\'.' ),
+	},
+	{
+		field: 'email_customer_new_account',
+		title: translate( 'New account' ),
+		subtitle: translate( 'Sent when customers sign up via checkout or account page.' ),
+	},
+];
 
 class Settings extends React.Component {
 
+	fetchSettings = ( props ) => {
+		const { siteId, fetchSettings } = props;
+		siteId && fetchSettings( siteId );
+	}
+
 	componentDidMount = () => {
-		const { siteId, fetchEmailSettings: fetch } = this.props;
-		siteId && fetch( siteId );
+		this.fetchSettings( this.props );
 	};
 
 	componentWillReceiveProps = newProps => {
-		if ( newProps.siteId === this.props.siteId ) {
-			return;
+		if ( newProps.siteId !== this.props.siteId ) {
+			this.fetchSettings( newProps );
 		}
+	};
+
+	notificationsToggle = () => {
+		undefined;
+	}
 
-		const { siteId, fetchEmailSettings: fetch } = newProps;
-		siteId && fetch( siteId );
+	recipientsChange = () => {
+		undefined;
+	}
+
+	renderOriginNotification = ( item, index ) => {
+		const { settings, loaded, loading } = this.props;
+		return <NotificationsOrigin
+			key={ index }
+			item={ item }
+			isPlaceholder={ loading }
+			recipient={ loaded ? settings.email[ item.field ] : '' }
+			onChange={ this.recipientsChange }
+		/>;
+	}
+
+	renderInternalNotification = ( item, index ) => {
+		const { settings, loaded, loading } = this.props;
+		return <InternalNotification
+			key={ index }
+			item={ item }
+			checked={ 'yes' === ( loaded && settings[ item.field ].enabled ) }
+			recipient={ loaded ? settings[ item.field ].recipient : '' }
+			isPlaceholder={ loading }
+			onToggle={ this.notificationsToggle }
+			onChange={ this.recipientsChange }
+		/>;
+	}
+
+	renderCustomerNotification = ( item, index ) => {
+		const { settings, loaded, loading } = this.props;
+		return <CustomerNotification
+			key={ index }
+			item={ item }
+			isPlaceholder={ loading }
+			checked={ 'yes' === ( loaded && settings[ item.field ].enabled ) }
+			onToggle={ this.notificationsToggle }
+		/>;
 	}
 
 	/* eslint-disable wpcalypso/jsx-classname-namespace */
 	render() {
 		return (
 			<div className="email-settings__container">
+				<ExtendedHeader
+					label={ translate( 'Origin' ) }
+				/>
+				<List>
+					{ originNotifications.map( this.renderOriginNotification ) }
+				</List>
+				<div>
+					<ExtendedHeader
+						label={ translate( 'Internal notifications' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **50** times:
`translate( 'notifications' )` *ES Score: 9*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/email-settings-ui)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).